staging/src/k8s.io/legacy-cloud-providers/azure/azure_storageaccount.go

```diff
@@ -119,6 +142,24 @@ func (az *Cloud) EnsureStorageAccount(accountOptions *AccountOptions, genAccount
 		}
 
 		if len(accountName) == 0 {
+			// set network rules for storage account
+			var networkRuleSet *storage.NetworkRuleSet
+			virtualNetworkRules := []storage.VirtualNetworkRule{}
+			for _, subnetID := range accountOptions.VirtualNetworkResourceIDs {
+				vnetRule := storage.VirtualNetworkRule{
+					VirtualNetworkResourceID: &subnetID,
+					Action:                   storage.Allow,
+				}
+				virtualNetworkRules = append(virtualNetworkRules, vnetRule)
+				klog.V(4).Infof("subnetID(%s) has been set", subnetID)
```

Comment:
nit: `subnetID(%s) has been set for storageAccount %s`

Reply:
that's for creating new storage account